api/envoy/config/bootstrap/v2/bootstrap.proto

```diff
@@ -19,7 +19,7 @@ import "envoy/config/trace/v2/trace.proto";
 import "envoy/config/metrics/v2/stats.proto";
 import "envoy/config/overload/v2alpha/overload.proto";
 import "envoy/config/ratelimit/v2/rls.proto";
-import "envoy/config/wasm/v2/wasm.proto";
+import "envoy/config/common/wasm/v2/wasm.proto";
```

Comment:
Alphabetical order is broken. Is there any tool to fix?

Reply:
`./tools/check_format.py fix`, but it doesn't seem to fix this one.

Reply:
I fixed this one manually, but imports are not sorted in protos.